src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$secondary-color: #616070;
+$button-color: #00acdc;
+$margin-center: 0 auto;
+$family-roboto: "Roboto", "Helvetica", sans-serif;
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Styling through `*` does not apply to variable parameters, so it is worth keeping similar styling in more general `scss` files.